books-api/Take Book Contract.md

```diff
@@ -0,0 +1,108 @@
+# Take Book Contract
+
+### POST /api/books/{id}/take
+
+Request body:
+```ts
+{
+  сopyId: 1,
+  takenAtUtc: `15.08.2025`
```

Comment:
This should not be sent from UI, it should be automatically populated on the API side at runtime. Again is it a date or an instant?